### ISSUE TITLE
Import JSON reads only first field

### ISSUE THREAD

**Issue by jfulem** [Fields2Cover/Fields2Cover#17]
`Parser::importJson` reads only the first field from the JSON. If the JSON (GeoJSON) contains more fields one would expect that all the fields will appear in the `F2CFields fields` (which is a vector of `F2CField`).
```json
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "properties": {
        "Name": "field1"
      },
      "geometry": {
        "type": "Polygon",
        "coordinates": [
          [
            [
              -90.13470527300802,
              41.46915182229183,
              0
            ],
            [
              -90.133492914607,
              41.47136257818229,
              0
            ],
            [
              -90.13591763140903,
              41.47259252972829,
              0
            ],
            [
              -90.13680276033013,
              41.47207804303187,
              0
            ],
            [
              -90.13774689788767,
              41.47240763655048,
              0
            ],
            [
              -90.1383852635944,
              41.472825655176955,
              0
            ],
            [
              -90.13910946003789,
              41.473830496641824,
              0
            ],
            [
              -90.13992485152885,
              41.474373104542266,
              0
            ],
            [
              -90.14028963198977,
              41.474135965355195,
              0
            ],
            [
              -90.14038619150844,
              41.47374207122346,
              0
            ],
            [
              -90.13723727820812,
              41.46921613624346,
              0
            ],
            [
              -90.13470527300802,
              41.46915182229183,
              0
            ]
          ]
        ]
      }
    },
    {
      "type": "Feature",
      "properties": {
        "Name": "field2"
      },
      "geometry": {
        "type": "Polygon",
        "coordinates": [
          [
            [
              -90.12925795209429,
              41.46795419481983,
              0
            ],
            [
              -90.1324947808007,
              41.46807214926753,
              0
            ],
            [
              -90.13247510406907,
              41.46895680085933,
              0
            ],
            [
              -90.13523969032265,
              41.46891994061772,
              0
            ],
            [
              -90.13611530660938,
              41.46819010351813,
              0
            ],
            [
              -90.13625304400279,
              41.46792470617217,
              0
            ],
            [
              -90.13622352884705,
              41.46778463491272,
              0
            ],
            [
              -90.134688740749,
              41.46554345360544,
              0
            ],
            [
              -90.1340197305524,
              41.46541812210039,
              0
            ],
            [
              -90.13400005374898,
              41.46502000863133,
              0
            ],
            [
              -90.13193399284772,
              41.4637887904667,
              0
            ],
            [
              -90.12940552784,
              41.46336854897521,
              0
            ],
            [
              -90.12925795209429,
              41.46795419481983,
              0
            ]
          ]
        ]
      }
    }
  ]
}
```

**Comment by Gonzalo-Mier** [comment]
Thanks! Fixed on [commit 6d6b195](https://github.com/Fields2Cover/Fields2Cover/commit/6d6b195bc393c0cbcc2b18d3f4b291cc17d998fe).
Now also supports if those Polygon have inner rings.

**Comment by jfulem** [comment]
Thanks.
